sqlserver/requirements.txt

```diff
@@ -2,7 +2,4 @@
 adodbapi==2.6.0.7
 pyro4==4.36
 
-
-# optional - SDK question here
-# Requires unixodbc on Linux
```

Comment:
could you keep this comment line (and make it more explicit, maybe: `Requires unixodbc on Linux, so only build on windows for now`)